### DISCUSSION TITLE
Show different form layout on ViewRecord and EditRecord page

### DISCUSSION THREAD

**Discussion by lucacastelnuovo** [filamentphp/filament#6390] [Help] [answered]
Is it possible to display a different form in the View and Edit page?

**Comment by danharrin** [comment, marked as answer]
Is it really completely different?

If it is a completely different form, you can define a `getFormSchema()` method individually on each page and return a form schema.

Or you can hide/show elements contextually in the resource form - https://filamentphp.com/docs/2.x/admin/resources/getting-started#hiding-components-contextually

Avoid duplicating code where possible

**Reply by lucacastelnuovo** [reply]
Thank you for the response, in this case I would like to show the data with a custom view when not editing and only display the inputs when editing.

**Reply by danharrin** [reply]
You can just set the `$view` property for that View page class then. You don't need a form for that

**Reply by lucacastelnuovo** [reply]
This does not quite workout in my case because I would still like to see the relationmanager

**Reply by danharrin** [reply]
You can copy the code from the vendor version of `view-record.blade.php` into your custom view and it will work

**Reply by lucacastelnuovo** [reply]
Awesome idea, thank you
